index.html

```diff
@@ -0,0 +1,163 @@
+                    </ul>
+                </nav>
+            </div>
+        </div>
+    </header>
+    <section class="hero-container">
+        <div class="container-herooo">
+            <div class="hero">
+                <div class="hero-image">
+                    <img src="./assets/images/theme11.png" class="theme" alt="theme">
+                </div>
+                
+                <div class="sub-theme">
+                    <p>We are a nonprofit organization whose goal is to build a pipeline to STEM fields (Science, Technology, Engineering, Math) for underprivileged and underrepresented groups to encourage and promote diversity.</p>
+                </div>
+                <div class="data-venue">
+                    <p class="time">2023.10.15(THU) ~ 16(FRI)</p>
+                    <p class="venue">@ National Museum of Kenya, Nairobi Centermore</p>
+                </div>
+            </div>
+        </div>
+        
+    </section>
+    <main>
+        <div class="program">
+        <div class="program-content">
+            <h3 class="title">Main Program</h3>
+            <p class="title-underline"></p>
+        </div>
+        <div class="program-details">
+            <div class="program-card">
+                <img src="./assets/icons/program_icon_01 1.png" class="program-icon" alt="image1"/>
+                <h3 class="program-text">Networking</h3>
+                <p class="program-cont">Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+            </div>
+            <div class="program-card">
+                <img src="./assets/icons/program_icon_02 1.png" class="program-icon" alt="image1"/>
+                <h3 class="program-text">Projects</h3>
+                <p class="program-cont">Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+            </div>
+            <div class="program-card">
+                <img src="./assets/icons/program_icon_03 1.png" class="program-icon" alt="image1"/>
+                <h3 class="program-text">opportunity</h3>
+                <p class="program-cont">Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+            </div>
+            <div class="program-card">
+                <img src="./assets/icons/program_icon_04 1.png" class="program-icon" alt="image1"/>
+                <h3 class="program-text">Scholarships</h3>
+                <p class="program-cont">Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+            </div>
+            <div class="program-card">
+                <img src="./assets/icons/program_icon_05 1.png" class="program-icon" alt="image1"/>
+                <h3 class="program-text">mentoring</h3>
+                <p class="program-cont">Lorem ipsum dolor sit amet, ipsum dolor sit amet</p>
+            </div>
+        </div>
+        <div class="frame">
+            <a href="#join" class="joinSummit">See the whole program</a>
+        </div>
+        </div>
+    </main>
```

Comment:
Nice display here 😉 but make sure to also use the background image to make this match the design better. It should have a background like this one 👇🏽 

![Screenshot 2023-03-18 at 14 29 53](https://user-images.githubusercontent.com/45739963/226128379-719e81b0-2ad7-47bb-929c-69f936bfc996.png)